A change in Qt 5.11_beta3 leads to errors

Arch GNU/Linux, `bionic` branch

```
CMake Error at src/CMakeLists.txt:108 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```

According to [the official Qt documentation](https://doc.qt.io/qt-5/cmake-manual.html) `qt5_use_modules` macro is obsolete 

Same as https://github.com/ubports/ubuntu-download-manager/issues/4

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -105,11 +105,10 @@ if (TRUST_STORE_MIR_AGENT_ENABLED)
     core/trust/mir/prompt_main.cpp
   )
 
-  target_link_libraries(trust-prompt Qt5::Core Qt5::Gui Qt5::Qml Qt5::Quick)
-
   target_link_libraries(
     trust-prompt
 
+    Qt5::Core Qt5::Gui Qt5::Qml Qt5::Quick
     ${Boost_LIBRARIES}
     ${PROCESS_CPP_LDFLAGS}
   )

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -105,7 +105,7 @@ if (TRUST_STORE_MIR_AGENT_ENABLED)
     core/trust/mir/prompt_main.cpp
   )
 
-  qt5_use_modules(trust-prompt Core Gui Qml Quick)
+  target_link_libraries(trust-prompt Qt5::Core Qt5::Gui Qt5::Qml Qt5::Quick)
 
   target_link_libraries(
     trust-prompt

